Dockerfile.template

```diff
@@ -0,0 +1,46 @@
+FROM node:%%NODE_VERSION%%-alpine
+
+# @see http://label-schema.org/rc1/
+LABEL maintainer="Phase2 <outrigger@phase2technology.com>" \
+  # Replacement for the old MAINTAINER directive has fragmented.
+  # "vendor" prevents CentOS from leaking through, the other is for tools integrations.
+  vendor="Phase2 <projectname@phase2technology.com>" \
```

Comment:
Is projectname a placeholder here?

Reply:
For the template that should not be a placeholder. For that matter, not based on CentOS.

* [ ] Clean up the labels for the Dockerfile.template